android/beagle/src/main/java/br/com/zup/beagle/android/data/PreFetchHelper.kt

```diff
@@ -42,8 +43,9 @@ internal class PreFetchHelper {
 
     private fun preFetch(rootView: RootView, route: Route) {
         if (route is Route.Remote && route.shouldPrefetch) {
+            if (route.url is Bind.Expression) return
```

Comment:
What do you think about adding some warnings to help the developer?
Search in the project the class called `BeagleMessageLogs`  add this method.
```
   fun expressionNotSupportInPreFetch() {
        BeagleLoggerProxy.warning("expression is not support in pre fetch")
    }
```
and call this function in 
```suggestion
            if (route.url is Bind.Expression) {
            BeagleMessageLogsexpressionNotSupportInPreFetch()
            return
            }
```